clio-util/src/main/scala/org/broadinstitute/clio/util/auth/AuthUtil.scala

```diff
@@ -53,10 +73,11 @@ object AuthUtil extends ModelAutoDerivation with LazyLogging {
     for {
       jsonPath <- Either.cond(
         Files.exists(serviceAccountPath),
-        serviceAccountPath,
-        new RuntimeException(
-          s"Could not find service account JSON at $serviceAccountPath"
-        )
+        serviceAccountPath, {
+          val message = s"Could not find service account JSON at $serviceAccountPath"
+          logger.warn(message)
```

Comment:
Oh wait, this is redundant with the existing log at [line 58](https://github.com/broadinstitute/clio/pull/183/files#diff-0d435faf7a01bc5ed20ddd17837dc161R58).